__tests__/app.test.js

```diff
@@ -26,4 +26,24 @@ describe('/api/topics', () => {
             })
         })
     });
+});
+
+describe('/api/articles/:article_id', () => {
+    test('GET request responds with an array of topic objects, each of which has slug and description properties', () => {
+        const article_id = 1
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(( { body }) => {
+            expect(body).toMatchObject({
```

Comment:
The body should not be the article itself - but an object with a key of `article`